packages/components/docs/QueryModel.md

```diff
@@ -10,13 +10,12 @@ retrieved by the server. You can manually instantiate a `QueryModel`, but you wi
 use the `withQueryModels` HOC to inject the needed QueryModel(s) into your component. To create a QueryModel you will need
 to define a `QueryConfig` object. At a minimum, your `QueryConfig` must have a valid SchemaQuery, but we also support
 many other attributes that allow you to configure the model before it is loaded, all of the attributes can be found
-on the [`QueryConfig` interface](https://labkey.github.io/labkey-ui-components/interfaces/queryconfig.html) at
-`src/QueryModel/QueryModel.ts`.
+on the [`QueryConfig` interface](../src/public/QueryModel/QueryModel.ts#L40).
```

Comment:
This link from the docs over to the src code is nice, but it is the only one in this doc file. Should we be linking more?  Maybe we should switch to making each of the headers into src code links (i.e. QueryMode, QueryConfig, withQueryModels, GridPanel, DetailPanel)?

Reply:
Also, there are still several links to "labkey.github.io" in the `QueryModel.ts` property docs I previously added. Those should be updated / removed as well.

Reply:
The labkey.github.io links are all for the api-js docs which I think is what we want. I may remove the formatting for Typedoc though. I will add links to source for the other components in QueryModel docs.

Reply:
Ah yes, the links to the api-js docs makes sense. I didn't look too closely at them, just did a text search in IntelliJ.